.github/workflows/conda.yml

```diff
@@ -68,8 +68,8 @@ jobs:
           APPLITOOLS_BATCH_ID: ${{ github.sha }}
           GITHUB_BRANCH_NAME: ${{ github.head_ref }}
         run: |
-          git clone https://github.com/mantidproject/mantidimaging-data.git ~/mantidimaging-data
-          wget -O ~/mantidimaging-data/24737.nxs https://github.com/DiamondLightSource/Savu/blob/master/test_data/data/24737.nxs?raw=true
+          wget https://github.com/mantidproject/mantidimaging-data/archive/refs/heads/master.zip
+          unzip master.zip
           xvfb-run --auto-servernum python -m pytest -p no:xdist -p no:randomly -p no:repeat -p no:cov mantidimaging/eyes_tests
```

Comment:
If you add an `-rs` argument to pytest it will show the reason for the skipped tests.